components/client/src/main/java/com/hotels/styx/client/StyxHttpClient.java

```diff
@@ -336,19 +335,27 @@ public String toString() {
      * A builder for {@link com.hotels.styx.client.StyxHttpClient}.
      */
     public static class Builder {
-        private final BackendService backendService;
+
+        private final Id applicationId;
         private MetricRegistry metricsRegistry = new CodaHaleMetricRegistry();
         private List<RewriteRule> rewriteRules = emptyList();
         private RetryPolicy retryPolicy = new RetryNTimes(3);
         private LoadBalancer loadBalancer;
         private boolean contentValidation;
         private OriginStatsFactory originStatsFactory;
         private String originsRestrictionCookieName;
+        private StickySessionConfig stickySessionConfig = stickySessionDisabled();
 
-        public Builder(BackendService backendService) {
-            this.backendService = checkNotNull(backendService);
+        public Builder(Id applicationId) {
```

Comment:
I think we should call the parameter and field `backendServiceId` for consistency.